src/terms.js

```diff
@@ -117,13 +118,16 @@ export default [
     'facilitat(e|or)',
     'fast track',
     'fault.tolerant',
+    'fintech',
     'first.to.market',
     'flexibility',
     'foot view',
+    'format agnostic',
```

Comment:
```suggestion
    'format.agnostic',
```

(to catch "format-agnostic")